custom_components/mail_and_packages/const.py

```diff
@@ -571,8 +574,18 @@
     "post_de_delivered": {},
     "post_de_packages": {},
     "post_de_tracking": {},
-}
-
+    # Ebay
+    "ebay_delivering": {
```

Comment:
Since eBay doesn't actually deliver packages we wouldn't add a "delivering" sensor.

Reply:
eBay does send out the delivery emails though, which I thought it would count here.

Reply:
eBay would fall under the same 'weirdness' that amazon does.

Reply:
Ah got it, thanks! Apologies for the inconvenience

Reply:
No need to apologize. Thanks for adding a new feature.